salt/utils/minions.py

```diff
@@ -257,7 +257,7 @@ def _check_cache_minions(self,
                              exact_match=False):
         '''
         Helper function to search for minions in master caches
-        If 'greedy' return accepted minions that matched by the condition or absend in the cache.
+        If 'greedy' return accepted minions that matched by the condition or absent in the cache.
```

Comment:
This could benefit from a slight rewrite:

`If 'greedy', then return accepted minions matched by the condition or those absent from the cache.`

Reply:
Good call, I've updated this docstring.